INSTALL.rst

```diff
@@ -1,6 +1,244 @@
 Installing scikit-image
 =======================
 
+We outline 4 different ways which we recommend using to get started with image
+processing and scikit-image.
+
+1. `Scientific Python Distributions`_ -- Recommended
+2. `pip based installation`_
+3. `conda based installation`_
+4. `System-wide package manager`_
+
+After your installation, you should be able to run the following code
+from a Python console or Jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+Should you have any questions getting started using ``scikit-image`` after
+reading this guide, please open an issue on
+`github <https://github.com/scikit-image/scikit-image/issues>`_
+to get additional help.
+
+For those attempting to build scikit-image from source, or trying contribute
+code back to scikit-image, we've provided development instructions below.
+
+Scientific Python Distributions
+-------------------------------
+
+``scikit-image`` comes pre-installed with several Python distributions.
+These have the advantage of comming pre-installed with many other useful
+packages for scientific computing and image analysis. Follow the instructions
+on the respective websites to get going. If you don't have python installed on
+your computer, these are the fastest way to get started with your project.
+We can recommend the following 3 distributions:
```

Comment:
```suggestion
As of June 2020, we recommend one of the following 3 distributions:
```

I like including dates in advice like this. When trawling the internet for posts, it's often hard to determine when a post was written, and the clearest indicator of whether the advice is stale is the written date. Even a "last updated" indicator on the page is not great because different parts of the page can get updated while specific bits go stale.